### ISSUE TITLE
Bewitchment would do so well in this pack! Also Thaumcraft 6 would be cool too!

### ISSUE THREAD

**Issue by AlmightyChan** [DarkPacks/SevTech-Ages#4037]
First of all Witchery was one of the most in depth mods ever in my opinion and since bewitchment is a spiritual successor with the same goal in mind I am excited to see where it goes. Next as far as "flow" goes if you are working through different ages witchery is a logical step along side mods like totemic and blood magic it just fits. Not to mention what it adds to the game mechanics wise, the best fun being had with friends by cursing them or even in single player being able to try many different involved rituals to accomplish different feats. Finally I'm not surprised its not in here (yet), being so new and all, however if you have ever played the original Witchery you should understand all it has to offer so I implore you, add this fantastic mod.

     On to Thaumcraft, arguably more well know then the mod above, and equally as fun I was actually surprised to not see it in here. With all the many layers it seemed logical to include, though I agree that the theme is a bit more outlandish then some of the other magic mods, still it would be a wonderful addition, adding a much appreciated extra layer of depth to the magic portion of the pack.

     Over all the pack you have is incredible and the amount of effort put into it is clear, I only suggest these mods as I think they would fit well and add even more amazing things to do in the world you have created.

Mod Details : Essentially a Witchery Mod successor, if Witchery weren't outdated I would suggest that instead.
Name: Bewitchment
CurseForge Link: https://www.curseforge.com/minecraft/mc-mods/bewitchment

Mod Details : I mean its Thaumcraft but updated still in beta but super well done.
Name: Thaumcraft 6
CurseForge Link: https://www.curseforge.com/minecraft/mc-mods/thaumcraft

**Comment by Darkosto** [comment]
I'm not familiar with Bewitchment at the moment so I don't have a comment on that one as of now. 

Having said that, I do have some problems with the current version of 1.12.2 Thaumcraft. It was supposed to be in SkyFactory 4 but we found some pretty glaring issues. One of those being, if the mod, for any reason, had to be removed it would corrupt your world and force you start again in a new world. This is definitely not a risk I'm willing to take.

**Comment by AlmightyChan** [comment]
Yeah that’s an understandable problem and as for Bewitchment, it might be worth checking out as long as it doesn’t cause problems as I would fit well into the pack

P.S. Currently playing your pack but we added dynamic trees and bewitchment and it seems to work well, as in no issues at the moment.
